.werft/build.js

```diff
@@ -284,6 +286,12 @@ async function deployToDev(version, workspaceFeatureFlags, dynamicCPULimits, reg
     try {
         shell.cd("chart");
         werft.log('helm', 'installing Gitpod');
+
+        if (gcpStorage) {
+            flags+=` -f ../.werft/values.dev.gcp-storage.yaml`
+            // Here we could also add the secret 'remote-storage-gcloud' to access the GCP bucket.
+            // For now, one had to create the secret with `kubectl apply ...` by hand after deployment.
```

Comment:
Do I understand this correctly that after the deployment one still has to create a secret themselves?
If so, can we not use one that's available to werft and copy that one over?

Reply:
We definitely can. And I have the proper code already at hand. What I am missing is the appropriate secret available in werft. I tried it with the following secret but that one apparently does not have the proper rights.

https://github.com/gitpod-io/gitpod/blob/15d59d9011b50db86ad6a357e1072b3dfcdea67d/.werft/build.js#L9

If we have proper secret available in werft or could easily add one we can change this quite simply. Otherwise, we could also move this to another PR.

Reply:
Moved to #3394